xml/System.ComponentModel/INotifyPropertyChanged.xml

```diff
@@ -390,7 +390,7 @@ End Class
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.ComponentModel.INotifyPropertyChanged.PropertyChanged> event can indicate all properties on the object have changed by using either `null` or <xref:System.String.Empty?displayProperty=nameWithType> as the property name in the <xref:System.ComponentModel.PropertyChangedEventArgs>.  
+ The <xref:System.ComponentModel.INotifyPropertyChanged.PropertyChanged> event can indicate all properties on the object have changed by using either `null` or <xref:System.String.Empty?displayProperty=nameWithType> as the property name in the <xref:System.ComponentModel.PropertyChangedEventArgs>.  Note that in a UWP application, String.Empty must be used rather than null.
```

Comment:
```suggestion
 The <xref:System.ComponentModel.INotifyPropertyChanged.PropertyChanged> event can indicate all properties on the object have changed by using either `null` or <xref:System.String.Empty?displayProperty=nameWithType> as the property name in the <xref:System.ComponentModel.PropertyChangedEventArgs>. Note that in a UWP application, <xref:System.String.Empty?displayProperty=nameWithType> must be used rather than `null`.
```